doc/man3/OSSL_print_attribute_value.pod

```diff
@@ -0,0 +1,55 @@
+=pod
+
+=head1 NAME
+
+OSSL_print_attribute_value
+- Print an X.500 directory attribute value
+
```

Comment:
Hmm, do you really intend to make it a public function? In that case it must be placed into openssl/x509.h and not crypto/x509.h

However I am not sure it should be.

Reply:
I am open to it being a private function, by the way. It just seemed kind of foundational to be able to print attribute values. What makes it preferable to be public or private? Is there some guiding principle for this?

Reply:
If there is a clear use case for 3rd party applications and the API is not controversial/unfinished by other means, it can be public.

Otherwise if you need something internally, by default an internal API should be added.

Reply:
Okay. Yeah, I do suspect this to be a controversial API. I imagine everybody has a different opinion on how these values should be printed. And I suspect this function will have a long tail of small tweaks throughout time. I'll make it private, since it matches both criteria very well. Thank you!

Reply:
Removed from the public API.